lib/dialects/abstract/query-generator.js

```diff
@@ -2245,6 +2248,10 @@ module.exports = (function() {
 
     booleanValue: function(value) {
       return value;
+    },
+
+    geometrySelect: function(column) {
```

Comment:
This function is never overridden by dialects, so it could just be inlined, right?